hack/cherry_pick_pull.sh

```diff
@@ -123,18 +123,31 @@ for pull in "${PULLS[@]}"; do
 done
 gitamcleanup=false
 
+function make-a-pr() {
+  echo "+++ Now you must propose ${NEWBRANCH} as a pull against ${BRANCH} (<--- NOT MASTER)."
+  echo "    You are constructing a pull against the upstream release branch! To do"
+  echo "    this in the GitHub UI, when you get to the 'Comparing Changes' screen, keep the"
+  echo "    'base fork' as GoogleCloudPlatform/kubernetes, but change the 'base' to e.g. 'release-1.0',"
+  echo "    presumably ${BRANCH}. (This selection is near the top left.)"
```

Comment:
The fixed text part is giving an example. The other part is taking a very solid guess at what you used as the thing you based this on, but in reality, if you know what you're doing, you can pass an awful lot of things to this script and it just works (and it's hard to stop you). So `${BRANCH}` could be anything from a random git commit, to a local branch, to someone's private remote branch, etc. I'm not saying you SHOULD use those, but you COULD.